app/src/js/reducers/workflows.js

```diff
@@ -3,19 +3,18 @@ import { set } from 'object-path';
 import {
   WORKFLOWS,
   WORKFLOWS_INFLIGHT,
-  WORKFLOWS_ERROR
+  WORKFLOWS_ERROR,
+  SEARCH_WORKFLOWS,
+  CLEAR_WORKFLOWS_SEARCH
 } from '../actions/types';
 import { createReducer } from '@reduxjs/toolkit';
 
 export const initialState = {
-  list: {
-    data: [],
-    meta: {},
-    params: {},
-    inflight: false,
-    error: false
-  },
-  map: {}
+  data: [],
```

Comment:
Why did you want to get rid of list?

Reply:
Why did we want to keep it?
It made for a deep structure and was somewhat confusing.  I thought flattening was just simplification.

Reply:
I guess if there's no reason to keep it. I just thought there was a reason all the reducers have the list structure. Did it have to do with going into Tables? or Lists?

Reply:
I did not need it.  And it makes the reducers slightly more complicated, so I got rid of it.  Easy enough to add back if we find it useful. But I'd prefer to wait unless we know a reason for the structure.

Reply:
I'm looking at this. And there's reasons, but they're not documented, and I don't fully understand them, and they don't work for workflow. I will ping you separately.

Reply:
I will add it back in this case.  There's a bunch of hidden assumptions for sure.

Reply:
d65552cb